packages/shared/locales/en.json

```diff
@@ -729,7 +729,8 @@
         "searchAgain": "Search again",
         "searching": "Searching...",
         "closeFirefly": "Close Firefly",
-        "generateAddress": "Generate address"
+        "generateAddress": "Generate address",
+        "migrateAgain": "Migrate again"
```

Comment:
Maybe it would be good not to name the button "Migrate again". This could be confusing that the migration was not successful. Maybe something like "Migrate next index" or "Next Migration" would be better.

Reply:
Good idea